src/javascript/actions/CreateNewContent/CreateNewContentDialog.jsx

```diff
@@ -0,0 +1,100 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {
+    Dialog,
+    DialogTitle,
+    DialogActions
+} from '@material-ui/core';
+import {Button} from '@jahia/design-system-kit';
+import {translate} from 'react-i18next';
+import {compose} from 'react-apollo';
+import {withStyles} from '@material-ui/core';
+
+import {TreeView} from '../../DesignSystem/TreeView';
+
+const treeProps = {
+    tree: [
+        {id: 'A',
+            label: 'A level1',
+            iconURL: 'https://image.flaticon.com/icons/svg/1973/1973617.svg',
+            childs: [
+                {id: 'A1', label: 'A-1 level2', iconURL: 'https://image.flaticon.com/icons/svg/1973/1973617.svg'},
+                {id: 'A2', label: 'A-2 level2'},
+                {id: 'A3', label: 'A-3 level2'},
+                {id: 'A4', label: 'A-4 level2'}
+            ]
+        },
+        {id: 'B',
+            label: 'B level1',
+            iconURL: 'https://image.flaticon.com/icons/svg/1973/1973617.svg',
+            childs: [
+                {id: 'B1', label: 'B-1 level2'},
+                {id: 'B2', label: 'B-2 level2'},
+                {id: 'B3', label: 'B-3 level2'},
+                {id: 'B4', label: 'B-4 level2', childs: [
+                    {id: 'B11', label: 'B-1-1 level3'},
+                    {id: 'B22', label: 'B-2-2 level3', iconURL: 'https://image.flaticon.com/icons/svg/1973/1973617.svg'},
+                    {id: 'B33', label: 'B-3-3 level3'},
+                    {id: 'B44', label: 'B-4-4 level3'}
+                ]}
+            ]
+        },
+        {
+            id: 'C',
+            label: 'C level1',
+            iconURL: 'https://image.flaticon.com/icons/svg/1973/1973617.svg',
+            childs: []
+        },
+        {
+            id: 'D',
+            label: 'D level1',
+            iconURL: 'https://image.flaticon.com/icons/svg/1973/1973617.svg'
+        }
+    ]
+};
+
+const styles = theme => ({
+    treeContainer: {
+        border: `1px solid ${theme.palette.ui.omega}`,
+        backgroundColor: '#F9FBFC',
```

Comment:
Please use theme variables for colors and sizes

Reply:
I cannot see it in figma :/

You are right, I will ask directly to the designer !